kchd/gpio.py

```diff
@@ -15,7 +15,6 @@ def __init__(self, leds: Set[KCHLED]) -> None:
 
         atexit.register(GPIO.cleanup)
         GPIO.setmode(GPIO.BCM)
-        GPIO.setwarnings(False)  # Ignore that the BOOT_20 and BOOT_40 LEDs were in use.
```

Comment:
We need to keep this as KCH intentionally takes over these LEDs.

When the updating animation is implemented, all of the boot LEDs will be used.